.spelling

```diff
@@ -527,6 +527,7 @@ Yessenov
 yournamespace
 Zack
 Zipkin
+RESPONSE_FLAGS
```

Comment:
Instead of putting this in the dictionary, please surround uses of this string with backticks

Reply:
Done.